ekf/tests/devekf.c

```diff
@@ -15,40 +15,73 @@
 
 #include <stdio.h>
 #include <unistd.h>
+#include <stdlib.h>
 
 #include <ekflib.h>
 
 #include "../tools/rotas_dummy.h"
 
-static void printUavVersors(quat_t *q, vec_t *start)
+
+enum printMode { prntVersor, prntAtt, printAcc };
+
+
+static void printUavVersors(ekf_state_t *uavState)
 {
+	vec_t start = { .x = uavState->enuX, .y = uavState->enuY, .z = uavState->enuZ };
 	vec_t x = { .x = 1, .y = 0, .z = 0 }, y = { .x = 0, .y = 1, .z = 0 }, z = { .x = 0, .y = 0, .z = 1 };
+	quat_t q = { .a = uavState->q0, .i = uavState->q1, .j = uavState->q2, .k = uavState->q3 };
+
+	quat_vecRot(&x, &q);
+	quat_vecRot(&y, &q);
+	quat_vecRot(&z, &q);
+
+	printf("%.3f %.3f %.3f %.3f %.3f %.3f %.3f %.3f %.3f %.3f %.3f %.3f\n", x.x, x.y, x.z, y.x, y.y, y.z, z.x, z.y, z.z, start.x, start.y, start.z);
+}
+
 
-	quat_vecRot(&x, q);
-	quat_vecRot(&y, q);
-	quat_vecRot(&z, q);
+static void printUavAtt(ekf_state_t *uavState)
+{
+	printf("YPR: %f %f %f YPR_DOT %f %f %f\n", uavState->yaw * 180 / 3.1415, uavState->pitch * 180 / 3.1415, uavState->roll * 180 / 3.1415, uavState->yawDot, uavState->pitchDot, uavState->rollDot);
+}
 
-	printf("%.3f %.3f %.3f %.3f %.3f %.3f %.3f %.3f %.3f %.3f %.3f %.3f\n", x.x, x.y, x.z, y.x, y.y, y.z, z.x, z.y, z.z, start->x, start->y, start->z);
+
+static void printUavAcc(ekf_state_t *uavState)
+{
+	printf("XYZ %f %f %f\n", uavState->accelX, uavState->accelY, uavState->accelZ);
 }
 
+
 int main(int argc, char **argv)
 {
 	ekf_state_t uavState;
-	quat_t q;
-	vec_t start = { 0 };
-	int i = 0;
+	enum printMode mode = prntVersor;
+
+	if (argc > 1) {
```

Comment:
Do you expect only one argument? If so, then:
```suggestion
	if (argc != 2) {
	    /* handle error */
	}
```